packages/gestalt/src/TableRowDrawer.js

```diff
@@ -50,11 +51,13 @@ export default function TableRowDrawer({ children, drawerContents, id }: Props):
   return (
     <Fragment>
       <tr aria-details={drawerContents ? id : undefined} ref={rowRef}>
-        {Number(stickyColumns) > 0 ? Children.map(children, renderCellWithAdjustedIndex) : children}
+        {Number(stickyColumns) > 0 ? Children.map(children, renderCellWithAdjustedIndex) : children}{' '}
+        {/* This needs to be fixed for children wrapped in React.Fragment when sticky columns are */}
+        present
```

Comment:
it looks like this word crept out of the comment

Reply:
YEP tahnks for catching this